spec/virtual_attributes_spec.rb

```diff
@@ -816,17 +816,26 @@ def lc
   end
 
   describe ".order" do
-    it "supports virtual attributes symbol" do
+    it "orders by virtual attributes symbol" do
       Author.order(:nick_or_name).first # fails
```

Comment:
Hey!  What the heck, man!  I definitely didn't authorize the use of my "name" in these tests!

Reply:
nick or name looks like a typo for Nick **for** name

also reminds me of trick or treat